server/config/appError.config.js

```diff
@@ -0,0 +1,16 @@
+
+class AppError extends Error {
+	constructor(message, statusCode = 500) {
+		super(message);
+		this.name = 'Stors';
```

Comment:
I think this should be "Storz" instead of "Stors"

Reply:
Okay.